include/libtorrent/config.hpp

```diff
@@ -386,10 +386,6 @@ POSSIBILITY OF SUCH DAMAGE.
 #define TORRENT_DEPRECATED
 #endif
 
-#ifndef TORRENT_USE_COMMONCRYPTO
-#define TORRENT_USE_COMMONCRYPTO 0
-#endif
-
```

Comment:
the benefit of definint macros to 0 or 1, instead of defining them or not defining them, is that the compiler can check spelling and warn if there's a mistake. Right now the convention is for configure macros set up by config.hpp are always defined, and only the ones from the outside are either defined or not. If anything, I would prefer to move towards every configuration macros always being defined to either 0 or 1 (unless you can think of a good reason not to)